src/app/components/pages/IsaacCompetition.tsx

```diff
@@ -1,24 +1,74 @@
 import React, { useEffect } from "react";
 import { SITE_SUBJECT_TITLE } from "../../services";
 import { TitleAndBreadcrumb } from "../elements/TitleAndBreadcrumb";
-import { Container } from "reactstrap";
+import "../../../scss/cs/competition.scss";
 
 export const IsaacCompetition = () => {
   useEffect(() => {
     document.title = "Isaac " + SITE_SUBJECT_TITLE;
   }, []);
 
   return (
-    <Container>
+    <>
       <TitleAndBreadcrumb currentPageTitle="Isaac Competition" breadcrumbTitleOverride="Isaac Competition" />
 
       <div id="section1">
         <h1>Section 1</h1>
       </div>
 
-      <div id="section2">
-        <h1>Section 2</h1>
-      </div>
-    </Container>
+      <section id="internetOfEverything" className="event-section">
```

Comment:
so does this not already exist as a component?

Reply:
No the main component within the events is the carousel.

Reply:
let's make this a new component then, it would massively simplify the the landing page component :)